app/views/shared/_base_page.html.erb

```diff
@@ -25,12 +24,8 @@
         content_item: content_item.to_h %>
     </div>
 
-    <% content_for :after_content do %>
```

Comment:
Shouldn't the footer content be after the "main" block, i.e. in the `:after_content` block?

Reply:
Uh, it's tricky! The contextual footer _should_ be in the main (it's related to the main content). Shared footers at the bottom of the page shouldn't be. I'll run through again just to make sure the footers are all in the right place.

Reply:
Have updated the contextual footers for case studies, travel advice, and take part (all of which were outside the main element but should have been inside).